scanpy/tests/highly_variable_genes.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import numpy as np
+import scanpy.api as sc
+from pathlib import Path
+
+FILE = Path(__file__).parent / Path('_scripts/seurat_hvg.csv')
+
+def test_higly_variable_genes_compare_to_seurat():
```

Comment:
Could you add the same test for the deprecated `filter_genes_dispersion`? We want to be sure that nobody touches it again in the future...

Thank you!

Reply:
Yes, ofc.